entityx/python/PythonSystem.h

```diff
@@ -10,8 +10,7 @@
 
 #pragma once
 
-// http://docs.python.org/2/extending/extending.html
-#include <Python.h>
```

Comment:
<boost/python.h> includes <Python.h> in under wrap_python.hpp:

https://github.com/boostorg/python/blob/boost-1.59.0/include/boost/python/detail/wrap_python.hpp

Including them both causes a macro conflicts in debug builds: (see http://www.boost.org/doc/libs/1_59_0/libs/python/doc/building.html#include-issues )